src/main/java/org/apache/solr/benchmarks/BenchmarksMain.java

```diff
@@ -147,15 +150,17 @@ public static void runIndexingBenchmarks(List<IndexBenchmark> indexBenchmarks, S
 		            if (setup.createCollection) {
 		            	log.info("Creating collection1: " + collectionName);
 		            	try {
-		            		solrCloud.deleteCollection(configsetName);
+                            solrCloud.deleteCollection(collectionName);
```

Comment:
just to make sure, it will not delete the collection from existing cluster? 
Also, it always creates new collection and delete that collection after test run?

Reply:
agree @patsonluk !

Reply:
@hiteshk25 I have added some experimental code with this approach https://github.com/fullstorydev/solr-bench/compare/patsonluk/external-forbid-collection-overwrite?expand=1 . However there are 2 concerns for this approach:
1. The code would not attempt to delete the collection if it exists, but it will still perform indexing benchmarking on it. This could be desirable in some cases (if the dev really want to run benchmark on some existing collection with other data), but could also be risky.
2. The cleanup would remove the test collection after the test. This could be a bit inconvenient if dev wants to look at the collection created as it always removes it afterwards now

Reply:
Looks bit tricky.
1. I ran into this collection exists problem, when tests fails and unable to delete the tasks.
2. For existing collections, one may just want query tests - no indexing tests.  I think for indexing test, it's better to create new test-collection.
3. I think collection can be re-used again, if created by test previously. can we figure this out?

Reply:
I was wondering if there's any way to "tag" a collection that it is for tests but I don't find any APIs for that. A rather naive solution is do something with the collection name - always add certain prefix/suffix, but that could be confusing as the collection name might end up be different from the one specified in the config

Reply:
Added a JIRA story about this https://fullstory.atlassian.net/browse/SAI-4325 to keep track of it